returnn/tf/engine.py

```diff
@@ -1520,6 +1526,7 @@ def train_epoch(self):
         batch_size=self.batch_size,
         max_seqs=self.max_seqs,
         max_seq_length=self.max_seq_length,
+        min_seq_length=self.min_seq_length,
```

Comment:
It's not used?

Reply:
what do you mean?

Reply:
Ah sorry, I didn't realized it is already implemented.